tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecCommonTools/SpecCommonTools.cs

```diff
@@ -4,18 +4,27 @@
 using System.Diagnostics;
 using ModelContextProtocol.Server;
 using Azure.Sdk.Tools.Cli.Helpers;
+using Azure.Sdk.Tools.Cli.Contract;
+using System.CommandLine;
 
 namespace AzureSDKDSpecTools.Tools
 {
     [Description("This type contains tools to run various common tasks in specs repo")]
     [McpServerToolType]
-    public class SpecCommonTools(IGitHelper _gitHelper, ILogger<SpecCommonTools> _logger)
+    public class SpecCommonTools(IGitHelper _gitHelper, ILogger<SpecCommonTools> _logger): MCPTool
     {
         private IGitHelper gitHelper = _gitHelper;
         private ILogger<SpecCommonTools> logger = _logger;
 
         static readonly string GET_CHANGED_TYPESPEC_PROJECT_SCRIPT = "eng/scripts/Get-TypeSpec-Folders.ps1";
 
+        // Commands
+        private const string getModifiedProjectsCommandName = "get-modified-projects";
+
+        // Options
+        private readonly Option<string> repoRootOpt = new(["-repo-root"], "Path to azure-rest-api-spec repo root") { IsRequired = true };
```

Comment:
nit: `--repo-root` or `-r` are usually the standards. Probably a typo?

Reply:
Typo